static/js/components/ChannelSidebar_test.js

```diff
@@ -13,17 +14,17 @@ describe("ChannelSidebar", () => {
   it("should render sidebar", () => {
     const channel = makeChannel()
     const wrapper = renderSidebar(channel)
-    const [description] = wrapper.find(ReactMarkdown)
-    assert.equal(description.props.source, channel.description)
+    const description = wrapper.find(ReactMarkdown)
+    assert.equal(description.props().source, channel.description)
```

Comment:
what's the reason for this change?

Reply:
Flow is unhappy with the `const [description] = wrapper.find(ReactMarkdown)` line since it doesn't think the `Wrapper` is indexable. https://github.com/facebook/flow/issues/1323

Reply:
hmmm well I suppose we can go with this, although personally I think if the code if valid then flow is wrong and the code shouldn't have to change (especially given that this is a test file, and flow is of somewhat limited value when writing tests)